src/chocolatey/infrastructure.app/commands/ChocolateyInfoCommand.cs

```diff
@@ -38,7 +38,10 @@ public override void ConfigureArgumentParser(OptionSet optionSet, ChocolateyConf
                 .Add(
                     "s=|source=",
                     "Source - Source location for install. Can use special 'windowsfeatures', 'ruby', 'cygwin', or 'python' sources. Defaults to configured sources.",
-                    option => configuration.Sources = option.UnquoteSafe())
+                    option => {
+                        configuration.Sources = configuration.ExplicitSources = option.UnquoteSafe();
+                        configuration.ListCommand.ExplicitSource = true;

```

Comment:
This is not for this PR, but I need to capture it so I don't forget: Chocolatey CLI 1.x does not have an `ExplicitSource` property. We will want to consider if that part is needed for fixing it in the support branch.